### PR TITLE
Correct PYODIDE_BASE_URL for  console.html in CircleCI artifacts

### PR DESCRIPTION
**Pull request by rth**
Closes https://github.com/pyodide/pyodide/issues/648

By adjusting PYODIDE_BASE_URL before artifacts are stored in CircleCI but after the workspace has been saved (which is used for S3 deployment), I think we should be able to use console.html from CircleCI artifacts without using local overrides in Chrome https://github.com/pyodide/pyodide/issues/1384#issuecomment-817411787

### CODE REVIEW

**Comment by rth** [comment]
No this still fails due to CORS when loading packages in `console.html`. I need to change the `PYODIDE_BASE_URL` to take into account the first redirect that happens when downloading packages.

File: .circleci/config.yml [hunk 1]
```diff
@@ -129,6 +129,9 @@ jobs:
           paths:
             - ./packages/.artifacts
             - ./build
+      - run:
+          name: Prepare CircleCI artifacts
+          command: PYODIDE_BASE_URL="$CIRCLE_BUILD_URL/artifacts/0/root/repo/build/" make build/console.html
```
Comment (hoodmane) [inline .circleci/config.yml:134]: I think it will work if you do:
```suggestion
          command: PYODIDE_BASE_URL="./" make build/console.html
```

Comment (rth) [reply .circleci/config.yml:134]: Yes of course that's the most straightforward fix. Thanks!

**Comment by hoodmane** [comment]
Could you restart the `build-packages` CI so we can test this?

**Comment by rth** [comment]
OK, this this now work https://217-140441556-gh.circle-artifacts.com/0/root/repo/build/console.html !

**Comment by hoodmane** [comment]
Now that this works we should probably add a suggestion somewhere to the dev docs.

**Pull request merged**